src/renderer/components/file.tsx

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import * as PapaParse from 'papaparse'
+import React, { useRef } from 'react'
+import Button from '@material-ui/core/Button'
+import CloudUploadIcon from '@material-ui/icons/CloudUpload'
+
+export interface IFileInfo {
+  name: string
+  size: number
+  type: string
+}
+
+interface UiFileProps {
+  onFileLoaded: any
```

Comment:
Don't add untyped code in shared component library. For now you can just move this inside your multisend app folder where you have more freedom on code style. 

If you want this to be a shared component it will need to be cleaned up and fully typed properly